elevation_mapping/config/sensor_processors/realsense_d435.yaml

```diff
@@ -1,11 +1,11 @@
-# Written by Marko Bjelonic.
+# Written by Lefteris kotsonis.
 # Modeling Realsense D435 Sensor Noise for Improved 3D Reconstruction and Tracking.
 sensor_processor/type: structured_light
-sensor_processor/cutoff_min_depth: 0.35
-sensor_processor/cutoff_max_depth: 3.0
-sensor_processor/normal_factor_a: 0.00241809
-sensor_processor/normal_factor_b: 0.00662547
-sensor_processor/normal_factor_c: 0.77199589
+sensor_processor/cutoff_min_depth: 0.4
```

Comment:
Is this the real value? Might be problematic for close obstacles.

Reply:
According to the [specs](https://click.intel.com/intelr-realsensetm-depth-camera-d435.html), the min. range is 0.2 m. This is also roughly what we experienced.

Reply:
The reason I put the min. range at 0.4 m was because I could not evaluate the sensor at any closer distances. At distances smaller than 0.4 m the edges of the target were not visible anymore because of the field of view limits and therefore I could not derive noise models for the lateral noise. In my experience too, the sensor did provide reliable data up to distances of about 0.2 m as you also mentioned above. It would therefore make sense to correct the min. range to 0.2 m.

Reply:
I agree. @lkotsoni Can you please set the min depth to 0.2 m. Thanks!